translate/filters/checks.py

```diff
@@ -2476,6 +2476,36 @@ def nplurals(self, unit):
 
         return True
 
+    @critical
+    def untranslated_plurals(self, unit):
+        """Check if any of the plural targets is translated while other isn't.
```

Comment:
Check that all plural forms are provided, OR Check that all plural forms are translated